IbisHardwareIGSIO/ibishardwareIGSIOsettingswidget.cpp

```diff
@@ -45,6 +54,9 @@ void IbisHardwareIGSIOSettingsWidget::UpdateUI()
     }
 
     ui->autoStartLastConfigCheckBox->setChecked( m_igsio->GetAutoStartLastConfig() );
+
+    // Display HW module log in the textEdit box
+    //ui->logTextEdit
```

Comment:
Do you intend to enable this commented code?

Reply:
I removed the comment